### PR TITLE
created page for managing types of buildings

### PR DESCRIPTION
**Pull request by Kuz1302**

### CODE REVIEW

**Comment by Kuz1302** [comment]
[2.5h - Create page for managing types of buildings](https://trello.com/c/EnotPMUb/105-25h-create-page-for-managing-types-of-buildings)

**Comment by zozich** [comment]
[2.5h - Create page for managing types of buildings](https://trello.com/c/EnotPMUb/105-25h-create-page-for-managing-types-of-buildings)

File: frontend/admin_panel/src/components/ManagingTypesOfBuildings/BuildingsList/BuildingItem/index.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  return {
+    buildingsListByName: state.buildings.buildingsListByName,
+    buildingTypes: state.buildings.buildingTypes,
+    changedBuildingsList: state.buildings.changedBuildingsList
+  }
+}
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    updateBuildingsList: (building, selectedTypes, buildingList, changedBuildings) => {
```
Comment (zozich) [inline frontend/admin_panel/src/components/ManagingTypesOfBuildings/BuildingsList/BuildingItem/index.js]: All this code including dispatch should be moved to action creator function. Only one function call should be present here

File: frontend/admin_panel/src/components/ManagingTypesOfBuildings/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+import SearchBar from './Searchbar'
+import BuildingsList from './BuildingsList'
+
+class ManagingBuildingsTypes extends Component {
+  render () {
+    return (
+      <div>
+        <SearchBar/>
```
Comment (zozich) [inline frontend/admin_panel/src/components/ManagingTypesOfBuildings/index.js]: We don't need a search bar on this page

**Pull request closed without merging**